fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HMSExternalTable.java

```diff
@@ -863,14 +865,20 @@ private long getRowCountFromFileList() {
             estimatedRowSize += column.getDataType().getSlotSize();
         }
         if (estimatedRowSize == 0) {
+            LOG.warn("Table {} estimated size is 0, return 0.", name);
             return 0;
         }
 
         int totalPartitionSize = partitionValues == null ? 1 : partitionValues.getIdToPartitionItem().size();
         if (samplePartitionSize != 0 && samplePartitionSize < totalPartitionSize) {
+            LOG.info("Table {} sampled {} of {} partitions, sampled size is {}",
+                    name, samplePartitionSize, totalPartitionSize, totalSize);
             totalSize = totalSize * totalPartitionSize / samplePartitionSize;
         }
-        return totalSize / estimatedRowSize;
+        long rows = totalSize / estimatedRowSize;
+        LOG.info("Table {} rows {}, total size is {}, estimatedRowSize is {}",
```

Comment:
better only to print the info under unhealthy cases, such as "size==0" cases, and avoid printing all.

Reply:
This function is only called while loading the cache, will not print this log after cache is available.